cmake/FindRadium.cmake

```diff
@@ -66,19 +70,33 @@ IF ( RADIUM_ROOT_DIR )
         ${RADIUM_ROOT_DIR}/lib
         )
 
+#    FIND_LIBRARY ( GLBINDING_LIBRARIES
+#            NAMES glbinding glbinding32.lib glbinding.lib libglbinding.a
+#            PATHS ${RADIUM_ROOT_DIR}/${CMAKE_BUILD_TYPE}/3rdPartyLibraries/lib
+#            ${RADIUM_ROOT_DIR}/lib
+#            )
+
     SET ( Radium_FOUND TRUE )
+    find_package(GlBinding REQUIRED)
```

Comment:
With linux, this lines cause: 
```
CMake Error at cmake/FindRadium.cmake:80 (find_package):
  By not providing "FindGlBinding.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "GlBinding", but CMake did not find one.

  Could not find a package configuration file provided by "GlBinding" with
  any of the following names:

    GlBindingConfig.cmake
    glbinding-config.cmake

```

Assimp and GLBinding does not seems to be included the same way (no `find_package` for assimp). Is there a specific reason for that ?